units/ArmBuildings/LandFactories/armaap.lua

```diff
@@ -32,7 +32,7 @@ return {
 		sightdistance = 312,
 		terraformspeed = 1000,
 		workertime = 200,
-		yardmap = "oooooooooooooooooooooooooooooooooooooooooooooooo",
+		yardmap = "yyyoooyyy yoooooooy yoooooooy ooooooooo ooooooooo ooooooooo ooooooooo yoooooooy yyoooooyy",
```

Comment:
this should be entirely `o`
if the hitbox occupies the spot so does the footprint
convert the hitbox shape to from `CylY` to box